### ISSUE TITLE
Challenge/Authenticate

### ISSUE THREAD

**Issue by beatgammit** [jcelliott/turnpike#33]

**Comment by beatgammit** [comment]
Work done here - 66e8638

@jcelliott - Is this done?

**Comment by jcelliott** [comment]
Not quite. The basic challenge/response mechanism is there, but we still need to implement the actual WAMP-CRA authentication. I'll probably finish it over the weekend.

**Comment by beatgammit** [comment]
Should probably hold off until https://github.com/tavendo/WAMP/issues/128 is resolved.
